prelude/__internal__/Random.mad

```diff
@@ -0,0 +1,480 @@
+import type { Maybe } from "Maybe"
+
+import Byte from "Byte"
+import ByteArray from "ByteArray"
+import Float from "Float"
+import Integer from "Integer"
+import { floor, mod } from "Math"
+import Tuple from "Tuple"
+
+import List from "./List"
```

Comment:
@aboeglin I think this line is incorrect relative to how we hook to prelude but when I think about it it breaks my brain. (Do we have to first publish the changes to the extant prelude and then update this line?)

Reply:
As said privately it is fine. But here I'd rather use "List". If you locally copy your new List module to the globally installed std lib then it should be recognized correctly.